jcommune-view/jcommune-web-controller/src/test/java/org/jtalks/jcommune/web/view/RssViewerTest.java

```diff
@@ -120,4 +109,17 @@ public void testRssFeed() throws Exception {
         verify(rssViewerMock).buildFeedItems(model, request, response);
     }
 
+    private boolean containtsOnlyValidXMLChars(String stringToValidate) {
+        String pattern = "[^"
```

Comment:
Why not simply iterate over the array of chars and not check if `stringToValidate.contains(character)`.